components/category-select-table.vue

```diff
@@ -39,14 +40,14 @@
 
             ...mapGetters([
 
-                "categories",
+                "getCategories",
```

Comment:
Just for the sake of simplicity - why include `get` in the name? Is this something we just went with from the Miro? The code using the returned value from the getter should not care that it's interacting with a getter.

Since `categories` is now a more complex field in the refactor, and we really want the names of the categories here how about naming this getter something like `categoryNames`?

Reply:
I think we can take a page out of the old Java OOP best practices and use get(for getters) or set(for mutators) prefixes. It has a nice ring to it but I agree on the 2nd point maybe rename it to getCategoryNames?

Reply:
I think I like the combo here. @jarmoza: I like that the methodname tells you in easy language what it does. I haven't found any official recommendations so far, but I like the thinking in this issue: https://github.com/vuestorefront/vue-storefront/issues/2069

Reply:
@surchs Seems like a good basis for a standard.

Reply:
Good point. I'm tracking this here now: https://github.com/neurobagel/annotation_tool/issues/287. For the refactor I would say we make the vuex method names consistent as we implement them